data/json/items/generic.json

```diff
@@ -2153,6 +2153,57 @@
     "volume": 4,
     "bashing": 4
   },
+  {
+    "type": "GENERIC",
+    "id": "soft_adobe_brick",
+    "category": "spare_parts",
+    "symbol": ",",
+    "color": "brown",
+    "name": "soft adobe brick",
+    "name_plural": "soft adobe bricks",
+    "description": "A compacted mass of soil and natural fibers, still too wet to build with.  Load it onto a pallet and leave it to dry.",
+    "price": 0,
+    "material": "soil",
+    "weight": 1750,
+    "volume": 3,
+    "bashing": 2,
+    "to_hit": -3
+  },
+  {
+    "type": "GENERIC",
+    "id": "adobe_brick",
+    "category": "spare_parts",
+    "symbol": ",",
+    "color": "yellow",
+    "looks_like": "brick",
+    "name": "adobe brick",
+    "name_plural": "adobe bricks",
+    "description": "A compacted mass of soil and natural fibers, baked dry enough to harden into a brick.",
+    "material": "soil",
+    "flags": "NONCONDUCTIVE",
+    "weight": 1500,
+    "volume": 3,
+    "bashing": 10,
+    "to_hit": -2,
+    "price": 2000,
+    "qualities": [ [ "HAMMER", 1 ] ]	
+  },
+  {
+    "type": "GENERIC",
+    "id": "mortar_adobe",
+    "symbol": "%",
+    "color": "brown",
+    "name": "adobe mortar",
+    "name_plural": "adobe mortar",
+    "description": "A thick, pasty mud, low in sand content to reduce crumbling once dry.  Used to glue larger, heavier pieces of mud and clay together.",
+    "category": "spare_parts",
+    "container": "bag_canvas",
+    "price": 200,
+    "material": "powder",
```

Comment:
Can you pick some other material for this? Powder material is flammable, and a mix of soil and water surely wouldn't be flammable.

Reply:
On it 👍